src/main/java/hattrickdata/Arena.java

```diff
@@ -0,0 +1,31 @@
+package hattrickdata;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Optional;
+
+@Getter
+@Builder
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
```

Comment:
See above if this could be Java records, or Kotlin.  Nice thing with Kotlin is that it mixes seemlessly with Java code.